.github/workflows/build-test-release.yml

```diff
@@ -0,0 +1,157 @@
+          esy install 
+          esy build          
+          
+      - name: bs-emotion-ppx test
+        working-directory: bs-emotion-ppx/test
+        run: |                    
+          esy install
+          esy build               
+                           
+      - name: Retrieve bs-emotion-ppx binary path
+        id: bs-emotion-ppx-path
+        working-directory: bs-emotion-ppx
+        shell: bash
+        run: |                    
+          if [ "${{ matrix.os }}" == "windows-latest" ]; then
+            BS_EMOTION_PPX_PATH=$( echo "$(esy x which bs-emotion-ppx)") 
+            BS_EMOTION_PPX_PATH=$( cygpath --mixed --absolute "$BS_EMOTION_PPX_PATH")           
+          else
+            BS_EMOTION_PPX_PATH=$(echo "$(esy x which bs-emotion-ppx)")
+          fi
+          mkdir upload 
+          cp $BS_EMOTION_PPX_PATH upload/
+          if [ "${{ matrix.os }}" == "ubuntu-latest" ]; then
+            mv package.json upload/package.json
+            mv README.md upload/README.md
+            mv postinstall.js upload/postinstall.js
+          fi
+          UPLOAD_PATH=$( echo "$(pwd)/upload")
+          if [ "${{ matrix.os }}" == "windows-latest" ]; then
+            UPLOAD_PATH=$( cygpath --mixed --absolute "$UPLOAD_PATH")
+          fi          
+          echo "UPLOAD_PATH: $UPLOAD_PATH"          
+          echo "##[set-output name=path;]$UPLOAD_PATH"          
+      
+      - name: Upload artifacts ${{ matrix.os }}        
+        uses: actions/upload-artifact@master
+        with:
+          name: ${{ matrix.os }}
+          path: ${{ steps.bs-emotion-ppx-path.outputs.path }}
+
+  publish:    
+    needs: build_and_test
+    if: github.event_name == 'release' && success()
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+          registry-url: "https://registry.npmjs.org"    
+
+      - name: Download linux artifacts        
+        uses: actions/download-artifact@master
+        with:
+          name: ubuntu-latest
+          path: release/linux
+
+      - name: Download macOS artifacts        
+        uses: actions/download-artifact@master
+        with:
+          name: macOS-latest
+          path: release/darwin
+
+      - name: Download windows artifacts
+        uses: actions/download-artifact@master
+        with:
+          name: windows-latest
+          path: release/windows
+
+      - name: Move artifacts
+        run: |          
+          mkdir -p release/bin
+          mv release/darwin/bs-emotion-ppx release/bin/bs-emotion-ppx-darwin-x64.exe
+          mv release/windows/bs-emotion-ppx.exe release/bin/bs-emotion-ppx-win-x64.exe
+          mv release/linux/bs-emotion-ppx release/bin/bs-emotion-ppx-linux-x64.exe
+          mv release/linux/package.json release/package.json
+          mv release/linux/postinstall.js release/postinstall.js
```

Comment:
Why do these files need to be copied only for linux?

Reply:
Basically, we use the linux os to create the final package for all OSes(Win, Mac, Linux). This can easily be Win or Mac but Linux is much easier for me.